numpy/core/tests/test_mem_policy.py

```diff
@@ -357,24 +357,36 @@ def test_new_policy(get_module):
     c = np.arange(10)
     assert np.core.multiarray.get_handler_name(c) == orig_policy_name
 
-def test_switch_owner(get_module):
+@pytest.mark.parametrize("policy", ["0", "1", None])
```

Comment:
I think there is a problem with pytest and os.environ on PyPy. In any case, let's not hold up this PR for that platform.
```suggestion
@pytest.mark.xfail(sys.implementation.name == "pypy",
                   reason=("bad interaction between getenv and "
                           "os.environ inside pytest"))
@pytest.mark.parametrize("policy", ["0", "1", None])
```

Reply:
May still print out noise, we could set the warning filter to "always", but probably doesn't matter.